src/System.Runtime.Extensions/tests/System/BitConverter.cs

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit;
+
+public static class BitConverterTests
+{
+    [Fact]
+    public static void DoubleToInt64Bits()
+    {
+        Double input = 123456.3234;
```

Comment:
As a note: I specifically left the Convert.* and BitConverter tests using the type names to ease readability seeing as the type names match the method names.

I'd be fine changing this, but like I said, we should do it consistently across the whole repo.  I won't make any related changes for now.